rector.php

```diff
@@ -0,0 +1,91 @@
+use Rector\DeadCode\Rector\PropertyProperty\RemoveNullPropertyInitializationRector;
+use Rector\DeadCode\Rector\Stmt\RemoveUnreachableStatementRector;
+use Rector\Php70\Rector\StaticCall\StaticCallOnNonStaticToInstanceCallRector;
+use Rector\Php74\Rector\Closure\ClosureToArrowFunctionRector;
+use Rector\Php74\Rector\Property\RestoreDefaultNullToNullableTypePropertyRector;
+use Rector\Php74\Rector\Ternary\ParenthesizeNestedTernaryRector;
+use Rector\Php81\Rector\Array_\FirstClassCallableRector;
+use Rector\Php81\Rector\FuncCall\NullToStrictStringFuncCallArgRector;
+use Rector\Php81\Rector\Property\ReadOnlyPropertyRector;
+use Rector\Php82\Rector\Class_\ReadOnlyClassRector;
+use Rector\Php82\Rector\Param\AddSensitiveParameterAttributeRector;
+use Rector\Php83\Rector\ClassMethod\AddOverrideAttributeToOverriddenMethodsRector;
+use Rector\Php84\Rector\Param\ExplicitNullableParamTypeRector;
+use Rector\TypeDeclaration\Rector\ArrowFunction\AddArrowFunctionReturnTypeRector;
+use Rector\TypeDeclaration\Rector\ClassMethod\ReturnNeverTypeRector;
+use Rector\TypeDeclaration\Rector\Closure\ClosureReturnTypeRector;
+use Rector\TypeDeclaration\Rector\Empty_\EmptyOnNullableObjectToInstanceOfRector;
+
+return RectorConfig::configure()
+    ->withCache('./.cache/rector', FileCacheStorage::class)
+    ->withPaths([
+        __DIR__ . '/public',
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+    ])
+    ->withConfiguredRule(AddSensitiveParameterAttributeRector::class, [
+        'sensitive_parameters' => [
+            'password',
+            'currentPassword',
+            'confirmPassword',
+            'confirmedPassword',
+            'plainTextPassword',
+            'newPassword',
+            'tincan_password',
+            'tincanPassword',
+            'secret',
+            'two_factor_secret',
+            'token',
+            'appKey',
+        ],
+    ])
+    ->withRules([
+        ParenthesizeNestedTernaryRector::class,
+        ExplicitNullableParamTypeRector::class,
+    ])
+    ->withSkip([
+        AddOverrideAttributeToOverriddenMethodsRector::class,
+        ArgumentAdderRector::class,
+        ClosureToArrowFunctionRector::class,
+        EmptyOnNullableObjectToInstanceOfRector::class,
+        FirstClassCallableRector::class,
+        NullToStrictStringFuncCallArgRector::class,
+        ReadOnlyClassRector::class,
+        ReadOnlyPropertyRector::class,
+        RemoveNullPropertyInitializationRector::class,
+        RemoveUnreachableStatementRector::class,
+        AddSensitiveParameterAttributeRector::class,
+        RemoveUnusedPublicMethodParameterRector::class,
+        RestoreDefaultNullToNullableTypePropertyRector::class,
+        ReturnNeverTypeRector::class,
+        StaticCallOnNonStaticToInstanceCallRector::class,
+        ClosureReturnTypeRector::class,
+        EncapsedStringsToSprintfRector::class,
+        AddArrowFunctionReturnTypeRector::class,
+    ])
+    ->withSkipPath('src/Tempest/Framework/Exceptions/')
```

Comment:
Due to the weird structure of the controller and file in this directory Rector removes 90% of the controller.
Ignoring the directory so we should not see weird errors or phpunit failures